LICENSE

```diff
@@ -1,21 +1,11 @@
-MIT License
 
-Copyright (c) 2019 F-G Fernandez
+Copyright (c) 2019-2020 Pyronear contributors.
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+This software is made available under the terms of *either* of the licenses:
+- CEA CNRS Inria Logiciel Libre License, v2.1: for more details, please see LICENSE.CeCILL
+- GNU General Public License v3: for more details, please see LICENSE.GPL
```

Comment:
The PR should be update to integrate the Affero reference:
GNU Affero General Public License v3: for more details, please see LICENSE.AGPL

Reply:
Thanks for catching these :)